index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+   <div class="videos">
+        <section class="video-section">
+            <video src="" controls width="1000"></video>
+            <article class="video-container">
+                 <div class="video-bottom-section">
+                     <a href="#">
+                        <img class="channel-icon" src=""/>
+                        <div class="video-details">
+                            <a href="#" class="video-title">Video Title</a>
+                            <a href="#" class="video-channel-name">Channel Name</a>   
+                            <div class="video-metadata">
+                                <span>12K views</span>
+                                •
+                                <span>1 week ago</span>  
+                            </div> 
+                        </div>
+                     </a>
+                 </div>   
+            </article>
+        </section>
+   </div>   
+</body>
+</html>
```

Comment:
I can see you've got the talent to become a great software engineer ! 
Just a few things you have to work on 💯 

- Looks good so far, just add more code to make your page look complete.

Reply:
- [ ] Please make sure to explain the reason 'why' you are suggesting and how to improve. Kindly be specific in this inline comment, try to tell a student how and where to improve. It will make your review :100: clear.